### PR TITLE
Add fix for typechecks

### PR DESCRIPTION
**Pull request by dualcnhq**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "test:watch": "tsc && concurrently -k -r \"tsc --watch --preserveWatchOutput\" \"wtr --watch\"",
     "deploy": "yarn run deploy:run -e $(git branch --show-current)",
     "deploy:run": "yarn run prepare:ghpages && touch ghpages/.nojekyll && yarn run deploy:gh",
-    "deploy:gh": "gh-pages -t -d ghpages -m \"Build for $(git log --pretty=format:\"%h %an %ai %s\" -n1) [skip ci]\""
+    "deploy:gh": "gh-pages -t -d ghpages -m \"Build for $(git log --pretty=format:\"%h %an %ai %s\" -n1) [skip ci]\"",
+    "typecheck": "node node_modules/typescript/bin/tsc --noEmit"
```
Comment (iisa) [inline package.json:21]: what's the difference with `npx tsp --noEmit`?

Comment (iisa) [reply package.json:21]: u think we should add typecheck to test script?

Comment (dualcnhq) [reply package.json:21]: `yarn tsc --noEmit` can be simplified to that - we can add it too in test script

Comment (iisa) [reply package.json]: pardon, `tsc` is already in `test` script.

Comment (dualcnhq) [reply package.json]: updated package.json

File: package.json [hunk 2]
```diff
@@ -13,11 +13,12 @@
     "lint": "eslint --ext .ts,.html . --ignore-path .gitignore && prettier \"**/*.ts\" --check --ignore-path .gitignore",
     "format": "eslint --ext .ts,.html . --fix --ignore-path .gitignore && prettier \"**/*.ts\" --write --ignore-path .gitignore",
     "circular": "madge --circular --extensions ts .",
-    "test": "tsc && yarn run lint && yarn run circular && wtr --coverage",
+    "test": "tsc && yarn typecheck && yarn run lint && yarn run circular && wtr --coverage",
```
Comment (iisa) [inline package.json:16]:
```suggestion
    "test": "tsc && yarn run lint && yarn run circular && wtr --coverage",
```

**Review by iisa — APPROVED** [review]
ok thank u!

I ran it locally and demo app works well

**Pull request merged**